src/car.cpp

```diff
@@ -154,7 +173,68 @@ void Car::move(vec2 off)
 	m_position.y += off.y;
 }
 
+void Car::set_lane(int n)

```

Comment:
just fyi, in the future I think we should use the `direction` enum for lanes instead of just a number. Ideally we'll program the cars so they move relative to their current position/orientation, so they won't need to know which lane they're in at all, but this is fine for now